### PR TITLE
update disclosure content

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

update disclosure content

**Preview:** https://hds-website-git-br-disclosure-hashicorp.vercel.app/utilities/disclosure

### :link: External links

Jira ticket: [HDS-127](https://hashicorp.atlassian.net/browse/HDS-1273)

### CODE REVIEW

File: website/docs/utilities/disclosure/partials/code/how-to-use.md [hunk 1]
```diff
@@ -26,4 +28,7 @@ To actually work, you need an interactive element that can trigger a custom even
 
 When the content is disclosed, the container can be closed in different way: toggling again the visibility via the button (`click` or `enter/return`), clicking outside of the content, or via the `esc` key.
 
-**Important:** The "content" is not positioned in any way in relation to the toggle: this responsibility is left to the consumers (eg by applying a `position: absolute` to a wrapper around the content that is passed to the `:content` block).
\ No newline at end of file
+!!! Warning
+
+The "content" is not positioned in any way in relation to the toggle: this responsibility is left to the consumers (eg by applying a `position: absolute` to a wrapper around the content that is passed to the `:content` block).
+!!!
```
Comment (MelSumner) [inline website/docs/utilities/disclosure/partials/code/how-to-use.md:34]:
```suggestion

!!!

```

File: website/docs/utilities/disclosure/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Usage
```
Comment (Dhaulagiri) [inline website/docs/utilities/disclosure/partials/guidelines/guidelines.md:1]: ditto here: having some heading so the sidecar will appear

File: website/docs/utilities/disclosure/partials/code/how-to-use.md [hunk 3]
```diff
@@ -26,4 +28,8 @@ To actually work, you need an interactive element that can trigger a custom even
```
Comment (KristinLBradley) [inline website/docs/utilities/disclosure/partials/code/how-to-use.md:28]: To me, the first example doesn't seem that helpful or necessary. I would just keep the 2nd example which includes a button.

**Review by KristinLBradley — APPROVED** [review]
Since this is an internal utility, I think this is ok for now but I would probably delete the first "How to use" example myself. (See other comment.)

File: website/docs/utilities/disclosure/partials/code/how-to-use.md (hunk 3 above)
Comment (Dhaulagiri) [reply website/docs/utilities/disclosure/partials/code/how-to-use.md:28]: agreed. i'll remove it

**Pull request merged**